app/src/main/java/com/asfoundation/wallet/wallet_validation/generic/PhoneValidationFragment.kt

```diff
@@ -69,6 +72,10 @@ class PhoneValidationFragment : DaggerFragment(),
     if (arguments?.containsKey(ERROR_MESSAGE) == true) {
       errorMessage = arguments?.getInt(ERROR_MESSAGE)
     }
+    if (arguments?.containsKey(PREVIOUS_CONTEXT) == true) {
+      previousContext = arguments?.getString(PREVIOUS_CONTEXT)
+    }
+
```

Comment:
In this case since you later do a null check and send empty string, maybe you can use here arguments.getString(PREVIOUS_CONTEXT,"") and then remove the "?" from String?